### PR TITLE
HDDS-4169. Fix some minor errors in StorageContainerManager.md

### PR DESCRIPTION
**Pull request by cku328**
## What changes were proposed in this pull request?

Fixed typo and removed duplicate title bar.

## What is the link to the Apache JIRA

https://issues.apache.org/jira/projects/HDDS/issues/HDDS-4169

## How was this patch tested?

It's not necessary.

### CODE REVIEW

**Review by cxorm — APPROVED** [review]
Thank you @cku328 for the fix.
The work looks good to me, +1.

File: hadoop-hdds/docs/content/concept/StorageContainerManager.md [hunk 1]
```diff
@@ -46,7 +46,7 @@ replicas. If there is a loss of data node or a disk, SCM
 detects it and instructs data nodes make copies of the
 missing blocks to ensure high availability.
 
- 3. **SCM's Ceritificate authority** is in
+ 3. **SCM's Certificate authority** is in
```
Comment (amaliujia) [inline hadoop-hdds/docs/content/concept/StorageContainerManager.md:49]: Should it be both upper case or lower case for the first character? 
E.g. either `Certificate Authority` or `certificate authority`?

Comment (cku328) [reply hadoop-hdds/docs/content/concept/StorageContainerManager.md]: Thanks @amaliujia for the suggestion. Both names sound good to me.
I'll update the patch based on your suggestion.

**Comment by amaliujia** [comment]
Thanks! LGTM

**Comment by cxorm** [comment]
The failed `build-branch / unit (pull_request)` seems not related the patch.
Let's commit it again.

**Comment by cku328** [comment]
Thanks @cxorm & @amaliujia for the review.
Now the CI check has passed.

**Comment by cxorm** [comment]
Thank you @amaliujia for reviewing this PR and @cku328 for contributing it.

**Pull request merged**